bitrise.yml

```diff
@@ -19,29 +19,37 @@ workflows:
         title: Step audit
         inputs:
         - content: stepman audit --step-yml=./step.yml
+    - script:
```

Comment:
please do this, this way: https://github.com/bitrise-steplib/steps-xamarin-archive/blob/master/bitrise.yml#L32